src/main/java/spoon/support/visitor/equals/CloneHelper.java

```diff
@@ -134,4 +136,24 @@
 	protected <T extends CtElement> void addClone(Map<String, T> targetMap, String key, T value) {
 		targetMap.put(key, clone(value));
 	}
+
+	private CtElement initialTopLevel;
+	private CtElement cloneTopLevel;
```

Comment:
What is purpose of these two members? They are actually useless - not accessible.